source/NVDAObjects/window/excel.py

```diff
@@ -1464,7 +1468,8 @@ def script_openDropdown(self,gesture):
 	@script(
 		# Translators: the description  for a script for Excel
 		description=_("Sets the current cell as start of column header"),
-		gesture="kb:NVDA+shift+c")
+		gesture="kb:NVDA+shift+c",
+		category=SCRCAT_EXCEL)

```

Comment:
Please ensure end brackets are placed on a new line

```suggestion
		category=SCRCAT_EXCEL
	)
```